config/config.inc.php

```diff
@@ -131,11 +131,15 @@
 
 /**
  * @deprecated since 1.7.7
+ *
+ * Can't remove it yet, because some native modules use it.
```

Comment:
I already tried to do it here - https://github.com/PrestaShop/dashtrends/pull/69, but I didn't want to loose time and make this PR stuck because of one constant. We can remove it in some other PR.

Reply:
Removed! :-)